language/use-cases/sql-code-generation/sql_code_generation_langchain.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p>I suggest moving this to be in the same code cell as <code>generation_prompt</code> function which depends on this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>